test/NuGet.Clients.FuncTests/NuGet.CommandLine.FuncTest/Commands/SignCommandTests.cs

```diff
@@ -0,0 +1,215 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Threading.Tasks;
+using FluentAssertions;
+using NuGet.Test.Utility;
+using Test.Utility.Signing;
+using Xunit;
+
+namespace NuGet.CommandLine.FuncTest.Commands
+{
+    [Collection("Sign Command Test Collection")]
+    public class SignCommandTests
+    {
+        private const string _packageAlreadySignedError = "Error NU5000: The package already contains a signature. Please remove the existing signature before adding a new signature.";
+
+        private SignCommandTestFixture _testFixture;
+        private TrustedTestCert<TestCertificate> _trustedTestCert;
+        private string _nugetExePath;
+        private string _timestamper;
+
+        public SignCommandTests(SignCommandTestFixture fixture)
+        {
+            _testFixture = fixture ?? throw new ArgumentNullException(nameof(fixture));
+            _trustedTestCert = _testFixture.TrustedTestCertificate;
+            _nugetExePath = _testFixture.NuGetExePath;
+            _timestamper = _testFixture.Timestamper;
+        }
+
+        [Fact]
```

Comment:
These should be CI facts since external users cannot run these

Reply:
Added CI fact to the test involving timestamping. others should be fine to run on local machines.